lib/SyTest/Federation/Room.pm

```diff
@@ -268,20 +268,28 @@ sub create_and_insert_event
 
    my ( $event, $event_id ) = $self->create_event( %fields );
 
-   $self->_insert_event( $event );
+   $self->insert_outlier_event( $event );
 
    $self->{prev_events} = [ $event ];
 
    return $event unless wantarray;
    return ( $event, $event_id );
 }
 
+=head2 insert_event
+
+   $event = $room->insert_event( $event );
+
+Inserts a new event into the database, updating prev_events.
```

Comment:
perhaps you could explain what prev_events here is, and why I should care. I think I'd read this description and expect it to update `prev_events` on the event itself, but that doesn't appear to be the case.

similarly below.